### ISSUE TITLE
visualize problem

### ISSUE THREAD

**Issue by cgllv** [Ryan-Zirui-Wang/farthest-point-sampling#2]
I use --manually_step=True to visualize sampling process, but when I press 'N', open3D break: 
o3d.visualization.draw_geometries_with_key_callbacks([pcd_all, pcd_selected], key_to_callback)
  File "main_sample.py", line 62, in fit_step_callback
    vis.update_geometry()
TypeError: update_geometry(): incompatible function arguments. The following argument types are supported:
    1. (self: open3d.open3d.visualization.Visualizer, arg0: open3d.open3d.geometry.Geometry) -> bool

Invoked with: VisualizerWithKeyCallback with name Open3D
I don't know what's wrong, I also modified the python code ord('N') to ord('n')，but when I press 'n', open3D window have no change, so, do you know what' the problem,does it cause by version difference? I use python=3.5.6 and open3D=0.9.0

**Comment by ziruiw-dev** [comment]
Hi @fengweigang,

I am not sure what reason causes this problem but I am using

- python 3.6.9
- numpy 1.17.2
- open3d 0.7.0.0

in my environment. 

I do remember that I picked open3d 0.7 instead of open3d 0.9 for some reason but I can't remember.

I just checked the open3d doc, there is an interface change on `update_geometry()` between 0.7 and 0.9, so I think using 0.7 should fix the problem.

Best,
Zirui
